_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -138,7 +138,7 @@ The `codec` determines how the `s3` source parses each Amazon S3 object. For inc
 
 ### `newline` codec
 
-The `newline` codec parses each single line as a single log event. This is ideal for most application logs because each event parses per single line. It can also be suitable for S3 objects that have individual JSON objects on each line, which matches well when used with the [parse_json]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/parse_json/) processor to parse each line.
+The `newline` codec parses each single line as a single log event. This is ideal for most application logs because each event parses per single line. It can also be suitable for S3 objects that have individual JSON objects on each line, which matches well when used with the [parse_json]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/parse-json/) processor to parse each line.
```

Comment:
We just renamed this file to `parse_json` in #7707. We need to use the underscores for the processor names for auto-generation.

Reply:
Ah, ok.